test/e2e/suite/issuers/acme/certificate/http01.go

```diff
@@ -547,97 +404,6 @@ var _ = framework.CertManagerDescribe("ACME Certificate (HTTP01)", func() {
 		err = f.Helper().ValidateCertificate(f.Namespace.Name, certificateName, validations...)
 		Expect(err).NotTo(HaveOccurred())
 	})
-
-	It("should obtain a signed certificate with a single IP Address from the ACME server", func() {
```

Comment:
question: has this test been moved to the conformance suite? I can't find it.

Reply:
@jakexks thx for spotting that that test is missing, should have been https://github.com/jetstack/cert-manager/blob/fa45e867f8383c543afd553ad121603638476a7b/test/e2e/suite/conformance/certificates/tests.go#L239-L267 without the "Common Name"
At some point, I added that test; but I must have lost it during rebase.

Reply:
I'll add it now